docs/design.md

```diff
@@ -0,0 +1,102 @@
+# Knitter Features and Design
+
+[TOC]
+
+## Brief introduction
+
+- Knitter is a container networking solution, especially for NFV senarios. With Knitter, developers can quickly develope/deploy NFV applications of telecommunication cloud native senarios. Knitter provides a consistent high-level network abstract model to solve the networking orchestration for containerized applications in different use cases.
+
+
+## Features
+
+- support for multiple networks/interfaces.
+	- in telecommunication senarios, service data traffic is generally separated into several network planes, e.g., control plane, data plane and management plane. Those types of traffic have different QoS requirements. Multiple network interfaces are needed to isolate the networks and provide QoS ability. The current Kubernetes network model doesn't support multiple networks while Knitter can meet the multiple networks requirements.
```

Comment:
senarios -> scenarios will be normally used IMO.
If you agree this, please sweep other words :)

Reply:
cool! done.